modules/react-components/src/components/avatar/user-avatar.tsx

```diff
@@ -169,22 +176,30 @@ export const UserAvatar: FunctionComponent<UserAvatarPropsInterface> = (
             hoverable
             open={ showPopup }
             trigger={ (
-                <Avatar
-                    avatar
-                    shape="circular"
-                    bordered={ false }
+                <AppAvatar
                     className={ classes }
-                    image={ resolveAvatarImage() }
+                    image={ resolveAvatarImage() 
+                        ? resolveAvatarImage() 
```

Comment:
Cant we use nullish coelescing operator here?

eg: resolveAvatarImage() ?? ( /* Your component */)

Reply:
ack

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/4666/commits/0b4097a6d779470206fbb7583ba59dbff180f1f7